src/precice/impl/ReadDataContext.cpp

```diff
@@ -0,0 +1,23 @@
+#include "ReadDataContext.hpp"
+
+namespace precice {
+namespace impl {
+ReadDataContext::ReadDataContext(
+    mesh::PtrData data,
+    mesh::PtrMesh mesh)
+    : DataContext(data, mesh)
+{
+}
+
+void ReadDataContext::configureMapping(MappingContext mappingContext, MeshContext meshContext)
+{
+  PRECICE_TRACE();
+  PRECICE_ASSERT(meshContext.mesh->hasDataName(getDataName()));
+  mesh::PtrData data = meshContext.mesh->data(getDataName());
+  PRECICE_ASSERT(data != _providedData);
```

Comment:
Can you add a message to what this means?

Reply:
Fixed in 14edb529f5dbe4048ef5c0c74f3446ecc454d283.